src/main/java/org/cryptomator/ui/error/ErrorController.java

```diff
@@ -95,13 +127,141 @@ public void copyDetails() {
 		Clipboard.getSystemClipboard().setContent(clipboardContent);
 
 		copiedDetails.set(true);
-		CompletableFuture.delayedExecutor(2, TimeUnit.SECONDS, Platform::runLater).execute(() -> {
-			copiedDetails.set(false);
-		});
+		CompletableFuture.delayedExecutor(2, TimeUnit.SECONDS, Platform::runLater).execute(() -> copiedDetails.set(false));
 	}
 
-	/* Getter/Setter */
+	private void loadHttpResponse(HttpResponse<InputStream> response) {
+		if (response.statusCode() == 200) {
+			Map<String, ErrorDiscussion> errorDiscussionMap = new Gson().fromJson(//
+					new InputStreamReader(response.body(), StandardCharsets.UTF_8),//
+					new TypeToken<Map<String, ErrorDiscussion>>() {
+					}.getType());
+
+			if (errorDiscussionMap.values().stream().anyMatch(this::isPartialMatchFilter)) {
+				Comparator<ErrorDiscussion> comp = this::compareExactMatch;
+				Optional<ErrorDiscussion> value = errorDiscussionMap.values().stream().min(comp//
+						.thenComparing(this::compareSecondLevelMatch)//
+						.thenComparing(this::compareThirdLevelMatch)//
+						.thenComparing(this::compareIsAnswered)//
+						.thenComparing(this::compareUpvoteCount));
+
+				if (value.isPresent()) {
+					matchingErrorDiscussion = value.get();
+					errorSolutionFound.set(true);
+				}
+			}
+		}
+	}
+
+	/**
+	 * Checks if an ErrorDiscussion object is a partial match based on the presence of the error code's method code in its title.
+	 *
+	 * @param errorDiscussion The ErrorDiscussion object to be checked.
+	 * @return A boolean value indicating if the ErrorDiscussion object is a partial match:
+	 * - true if the object's title contains the error code's method code,
+	 * - false otherwise.
+	 */
+	public boolean isPartialMatchFilter(ErrorDiscussion errorDiscussion) {
```

Comment:
Method names don't usually are nouns ("Filter"). I'd suggest:
```suggestion
	public boolean containsMethodCode(ErrorDiscussion errorDiscussion) {
```

Reply:
method has been renamed